### PR TITLE
feat: create integrations in UI

### PR DESCRIPTION
**Pull request by asdfryan**
Also:
- added loading states in various related pages
- changed redirects to already include applicationId scoping for faster redirects

https://www.loom.com/share/a026a7eec0d4465f885a7270fcf146f0

### CODE REVIEW

File: apps/mgmt-ui/src/client.ts [hunk 1]
```diff
@@ -53,6 +53,23 @@ export async function deleteWebhook(applicationId: string): Promise<void> {
...
 
+export async function createRemoteIntegration(
+  applicationId: string,
+  data: CRMIntegrationCreateParams
+): Promise<Integration> {
+  const result = await fetch(`/api/internal/integrations/create`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'x-application-id': applicationId,
```
Comment (albertyfwu) [inline apps/mgmt-ui/src/client.ts:64]: who checks that the user is allowed to create integrations for this `applicationId`? This is in the FE code and not in the next.js lambdas. Can't anybody hit this endpoint, pass in an applicationId, and do this even for orgs they don't own?

File: apps/mgmt-ui/src/components/configuration/IntegrationCard.tsx [hunk 2]
```diff
@@ -40,7 +53,7 @@ export default function IntegrationCard(props: { integration: Integration; integ
           <Button
             variant="text"
             onClick={() => {
-              router.push(`/configuration/integrations/${category}/${providerName}`);
+              router.push(`/applications/${applicationId}/configuration/integrations/${category}/${providerName}`);
```
Comment (albertyfwu) [inline apps/mgmt-ui/src/components/configuration/IntegrationCard.tsx:56]: in the future maybe we can create another hook that automatically prepends this for simplicity

File: apps/mgmt-ui/src/client.ts (hunk 1 above)
Comment (asdfryan) [reply apps/mgmt-ui/src/client.ts:64]: There will be a follow-up PR

Comment (albertyfwu) [reply apps/mgmt-ui/src/client.ts:64]: https://github.com/supaglue-labs/supaglue/issues/465

**Review by albertyfwu — APPROVED** [review]

**Pull request merged**